### ISSUE TITLE
weighted cross entropy implementation

### ISSUE THREAD

**Issue by purplebutterflyy** [toonvds/CostSensitiveLearning#2]
Hi, thanks for sharing your code it's really helpful!
This is not an issue but rather a question. I see that you implemented the weighted cross entropy from scratch for the tested algorithms. Why is that? How is it any different than using sklearn's class weights for logistic regression for example?
Thanks.

**Comment by toonvds** [comment]
Happy to hear that our code is helpful!

For logistic regression, you could indeed use sklearn's `class_weight` or `sample_weight` argument. That approach should give the same results as our version.

Our approach allowed us to fairly compare different loss functions, which were not all supported by sklearn (specifically, the average expected cost).

**Comment by purplebutterflyy** [comment]
Makes sense, thanks for responding!
